spec/map-prototype-reduce.html

```diff
@@ -0,0 +1,22 @@
+<p>When the `reduce` method is called, the following steps are taken:</p>
+
+<emu-alg>
+    1. Let _map_ be the *this* value.
+        1. If Type(_map_) is not Object, throw a *TypeError* exception.
+        2. If _map_ does not have a [[MapData]] internal slot, throw a *TypeError* exception.
+    2. If IsCallable(_callbackfn_) is false, throw a *TypeError* exception.
+    3. Let _entries_ be the List that is _map_.[[MapData]].
+    4. Let _accumulator_ be *undefined*
```

Comment:
this is different from arrays, where omitting the initial value uses the first item in the iteration. why?

Reply:
Nope, see the loop.

Reply:
aha, thanks.